### DISCUSSION TITLE
SFP conflicts with the current method of reverting the new June UI

### DISCUSSION THREAD

**Discussion by Mantha065** [PhantomGamers/SFP#164] [General]
To bring everyone up to speed: The new June UI required a new way to revert to the old UI. People found a way, but recent Steam updates broke Friends from working on reverted UIs, meaning it would never connect to the Steam Friends servers. So a *new* way had to be found to revert the UI *and* keep Friends working, which was eventually found. However, I still need to change the UI skin (to OldGlory by Jonius7). To do that I ran SFP. 
To my horror, I found that running SFP now does...something...that ends up un-reverting the UI and restoring it to the current UI, which forces me to have to do the revert process all over again. I don't know what causes it, but running SFP will undo the UI reverting process, so while I am able to use the old UI, I can't use skins on the old UI.

For reference, here is a link to the current method of reverting to the old UI (and keeping Friends functional), which I compiled from a massive clusterfuck of "do this"/"no don't do that, do this".
https://steamcommunity.com/discussions/forum/0/6516193260168294059/?tscn=1695626195#c3875967149487755456

[SFP.log](https://github.com/PhantomGamers/SFP/files/12712284/SFP.log)

And then here is the most recent log from SFP, if it helps.

**Comment by PhantomGamers** [comment]
Are you putting `-vgui -nofriendsui` in the launch arguments field within SFP's settings?

**Reply by Mantha065** [reply]
I haven't touched SFP's settings, no.

**Reply by Mantha065** [reply]
Oh, should I have created this as an Issue, or is a discussion thread fine? I rarely use Github lol.

**Reply by PhantomGamers** [reply]
This was fine

**Comment by Mantha065** [comment]
I just now noticed the recent update to 0.0.52, and tried it out. I'm still getting the same problem as above. SFP just reverses whatever changes I've made to the Steam client and turns it back into the exact UI I'm trying to get away from.

**Reply by PhantomGamers** [reply]
Same solution as above:

> Are you putting -vgui -nofriendsui in the launch arguments field within SFP's settings?
